### DISCUSSION TITLE
It seems no /src/prob folder

### DISCUSSION THREAD

**Discussion by sm-liang** [PrincetonUniversity/athena#548] [Q&A] [answered]
I want try some write a new problem, according the wiki guide, I should use the /src/prob/ file, but I can't find it in my athena++ folder, I also try in code page of here, but I still not found.

**Comment by c-white** [comment, marked as answer]
That's a typo -- it should be `src/pgen/`. I'll fix it in the wiki soon.

**Reply by sm-liang** [reply]
Thank you for your explanation, CHris.

**Reply by felker** [reply]
I fixed the 1x occurrence @c-white

**Reply by c-white** [reply]
@felker Thanks, that's the one occurrence I know of.
